src/Mod/Path/Path/Base/Util.py

```diff
@@ -162,14 +161,14 @@ def clearExpressionEngine(obj):
 
 def toUnicode(string):
     """toUnicode(string) ... returns a unicode version of string regardless of the python version."""
-    return six.text_type(string)
+    return str(string)
```

Comment:
Better to eliminate the toUnicode function entirely--if you don't want to do it here, either leave a TODO or file an issue so we don't forget.

Reply:
done